src/views/text_area.rs

```diff
@@ -362,6 +362,11 @@ impl TextArea {
 }
 
 impl View for TextArea {
+    fn required_size(&mut self, constraint: Vec2) -> Vec2 {
+        self.compute_rows(constraint);
+        Vec2::new(1, self.rows.len())
+    }
```

Comment:
If the constraint sets a maximum width, then the line wraps are justified, aren't they?